Assets/Scripts/Gameplay/ConnectionManagement/ConnectionState/ClientConnectedState.cs

```diff
@@ -0,0 +1,43 @@
+using System;
```

Comment:
I don't see any meta files for this connection states, did you forget to commit them?

Reply:
nvm, I think I had them filtered out lol, resolving
silly sam